pkg/sdk/logging/api/v1beta1/fluentbit_types.go

```diff
@@ -208,6 +208,8 @@ type BufferStorage struct {
 	StorageBacklogMemLimit string `json:"storage.backlog.mem_limit,omitempty"`
 	// Available in Logging operator version 4.4 and later. If the `http_server` option has been enabled in the main Service configuration section, this option registers a new endpoint where internal metrics of the storage layer can be consumed. (default:Off)
 	StorageMetrics string `json:"storage.metrics,omitempty"`
+	// If the input plugin has enabled filesystem storage type, this property sets the maximum number of Chunks that can be up in memory. This is the setting to use to control memory usage when you enable storage.type filesystem. (default: 128)
+	StorageMaxChunksUp string `json:"storage.max_chunks_up,omitempty"`
```

Comment:
Shouldn't it be `int`?

Reply:
it doesn't add much in this case, but you are right, I changed it to int